.github/workflows/build-binaries-and-brew.yaml

```
@@ -146,15 +165,16 @@ jobs:
       - name: Download Linux artifact
         uses: actions/download-artifact@v4
         with:
-          name: holmes-ubuntu-20.04-${{ github.ref_name }}
+          name: holmes-ubuntu-22.04-${{ github.ref_name }}
       - name: Calculate hash
         id: calc-hash
-        run: echo "::set-output name=LINUX_BUILD_HASH::$(sha256sum holmes-ubuntu-20.04-${{ github.ref_name }}.zip | awk '{print $1}')"
+        run: echo "::set-output name=LINUX_BUILD_HASH::$(sha256sum holmes-ubuntu-22.04-${{ github.ref_name }}.zip | awk '{print $1}')"
```

actionlint (1.7.9)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)